spec/features/course_pages_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe 'CoursePages', type: :feature do
```

Comment:
Not the whole 'as a <user>, I can <do something> so that <reason>', something shorter would do, maybe 'Courses management'